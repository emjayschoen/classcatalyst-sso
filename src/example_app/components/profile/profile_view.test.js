import React from 'react';
import { shallow } from 'enzyme';
import ProfileView from './profile_view';

describe('<ProfileView />', () => {
  let subject;
  beforeEach(() => {
    subject = shallow(<ProfileView handleSubmit={jest.fn()} />);
  });

  it('renders correctly', () => {
    expect(subject).toMatchSnapshot();
  });
});
