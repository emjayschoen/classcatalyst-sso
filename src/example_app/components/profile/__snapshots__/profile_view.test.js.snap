// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProfileView /> renders correctly 1`] = `
<styled.div>
  <h1>
    My Account
  </h1>
  <form
    onSubmit={[MockFunction]}
  >
    <Field
      component={[Function]}
      label="First Name"
      name="firstName"
      type="text"
    />
    <Field
      component={[Function]}
      label="Last Name"
      name="lastName"
      type="text"
    />
    <styled.button
      buttonType="primary"
      disabled={false}
      primary={true}
      type="submit"
    >
      Submit
    </styled.button>
  </form>
</styled.div>
`;
